src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.wasm.cs

```diff
@@ -45,6 +45,16 @@ public TextBlock() : base("p")
 			OnTextAlignmentChangedPartial();
 			OnTextWrappingChangedPartial();
 			OnIsTextSelectionEnabledChangedPartial();
+			InitializeDefaultValues();
+
+		}
+
+		/// <summary>
+		/// Set default properties to align them with uwp.
+		/// </summary>
+		private void InitializeDefaultValues()
+		{
+			this.SetValue(VerticalAlignmentProperty, VerticalAlignment.Top, DependencyPropertyValuePrecedences.DefaultValue);
```

Comment:
This is not correct, the default `VerticalAlignment` is `Stretch` for `TextBlock` as for other FrameworkElement types. Can you share more information on why you added this code?